packages/web-components/package.json

```diff
@@ -86,8 +88,10 @@
     "start": "yarn start-storybook -p 6006 --docs",
     "start-storybook": "storybook dev",
     "build-storybook": "storybook build -o ./dist/storybook --docs",
+    "build-harness": "vite build test/harness",
+    "serve-harness": "vite preview test/harness",
     "e2e": "playwright test",
-    "test:dev": "playwright test"
+    "test:dev": "playwright test --ui"
```

Comment:
**suggestion:** monorepo uses `e2e:local` task name for this, I'd suggest to do the switch now. 

in upcoming weeks we will completely switch to nx project crystal which generates project task/targets behind the scenes (TL;DR all package.json#scripts will be gone and dissalowed to use )

Reply:
also FYI: future wise, existing VR test solution will use only storybook play apis

Reply:
I updated this command to `e2e:local` and added a script to handle switching between CI mode and UI mode: https://github.com/microsoft/fluentui/pull/33023/files#diff-6dfa7541b41973aaa309c3b33f93661bd250ca5638d674989b276d16d8a7bba9